modules/history/src/views/full/message.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import style from './style.scss'
+
+import { GoX } from 'react-icons/go'
+import { MdFileDownload } from 'react-icons/md'
+import { FiLink } from 'react-icons/fi'
+import { MdSearch } from 'react-icons/md'
```

Comment:
When you extract icons from the same package, you can group them `import { MdFileDownload, MdSearch } from 'react-icons/md'`